command/operator_migrate.go

```diff
@@ -144,8 +144,13 @@ func (c *OperatorMigrateCommand) Run(args []string) int {
 	}
 	c.logger = logging.NewVaultLogger(log.LevelFromString(c.flagLogLevel))
 
-	if (c.flagStart != "") && c.flagParallel != 1 {
-		c.UI.Error("Error: flag -start and -parallel can't be used together")
+	if c.flagParallel < 1 || c.flagParallel > 2147483647 {
```

Comment:
Can we use `math.MaxInt32` instead of a magic number (here and below)?

Reply:
As a matter of fact, we'll likely not gain much from running `math.maxInt32` goroutines in parallel and it may not even be possible. How do you feel about limiting `flagParallel` to some lower number. Say, 100?

Reply:
This should actually be fine as just `math.maxInt32`. We can leave it up to the user to properly tune, but maybe we should specifically call that out in the [docs](https://github.com/hashicorp/vault/blob/main/website/content/docs/commands/operator/migrate.mdx#usage).